features/networking/multus.feature

```diff
@@ -416,4 +416,248 @@ Feature: Multus-CNI related scenarios
+      | ["metadata"]["name"] | pod3-vlan200 |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"]| bridgevlan200 |
+      | ["spec"]["nodeSelector"]["test"] | worker1 |
+    Then the step should succeed
+    And the pod named "pod3-vlan200" becomes ready
+    And evaluation of `pod.name` is stored in the :pod3 clipboard
+    And I execute on the pod:
+      | ifconfig | net1 |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]` is stored in the :pod3_net1_ip clipboard
+    
+    #Clean-up required to erase bridge interfcaes created on node
+    Given I register clean-up steps:
+    """
+    the bridge interface named "mybridge" is deleted from the "<%= cb.nodes[0].name %>" node
+    the bridge interface named "mybridge.100" is deleted from the "<%= cb.nodes[0].name %>" node
+    the bridge interface named "mybridge.200" is deleted from the "<%= cb.nodes[0].name %>" node
+    """  
+    #making sure the pods in same vlan can communicate but in different vlans cannot
+    When I execute on the "<%= cb.pod1 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod2_net1_ip %> |
+    Then the step should succeed
+
+    When I execute on the "<%= cb.pod2 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod1_net1_ip %> |
+    Then the step should succeed
+    
+    When I execute on the "<%= cb.pod1 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod3_net1_ip %> |
+    Then the step should fail
+
+    When I execute on the "<%= cb.pod3 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod1_net1_ip %> |
+    Then the step should fail
+    
+    When I execute on the "<%= cb.pod3 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod2_net1_ip %> |
+    Then the step should fail
+    
+    When I execute on the "<%= cb.pod2 %>" pod:
+      | ping | -c1 | -W2 | <%= cb.pod3_net1_ip %> |
+    Then the step should fail
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24607
+  @admin
+  Scenario: macvlan plugin without master parameter	
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def without master pmtr via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-conf-without-master.yaml |
+      | n | <%= project.name %>                                                                                                                                    |
+    Then the step should succeed
+    
+    #Creating a pod absorbing above net-attach-def
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-conf |
```

Comment:
pod perhaps needs namespace also set, otherwise it will go into `default` project